src/components/Box/Box.tsx

```diff
@@ -26,14 +44,115 @@ type Props = {
    * 幅を指定。他のスタイルの影響を受け、幅が100%とならない場合にのみ使用
    */
   width?: 'full';
+  /**
+   * 内包するテキストをボールドとするかどうか。指定しない場合は親要素のスタイルを継承、trueでボールド、falseとするとnormal
+   */
+  textBold?: boolean;
+  /**
+   * 文字色の抽象値。のかのtext系Propとは独立して指定可能
+   */
+  textColor?: TextColor;
 } & PaddingProps &
   MarginProps &
   RadiusProp;
 
+type PropsWithoutText = BaseProps & {
+  /**
+   * 配下に含むテキストの種類
+   */
+  textType?: undefined;
+  /**
+   * 配下に含むテキストのフォントサイズの抽象値。合わせてtextTypeの指定が必須で、typeに応じた値が指定可能
+   */
+  textSize?: never;
+  /**
+   * 配下に含むテキストの行送りの抽象値（`line-height`）。合わせてtextTypeとtextSizeの指定が必須で、typeに応じた値が指定可能
+   */
+  textLeading?: never;
+};
+
+type PropsWithTextBody = BaseProps & {
+  /**
+   * 配下に含むテキストの種類
+   */
+  textType: Extract<TextType, 'body'>;
+  /**
+   * 配下に含むテキストのフォントサイズの抽象値。合わせてtextTypeの指定が必須で、typeに応じた値が指定可能
+   */
+  textSize?: BodyFontSize;
+  /**
+   * 配下に含むテキストの行送りの抽象値（`line-height`）。合わせてtextTypeとtextSizeの指定が必須で、typeに応じた値が指定可能
+   */
+  textLeading?: BodyLeading;
+};
+
+type PropsWithTextNote = BaseProps & {
+  /**
+   * 配下に含むテキストの種類
+   */
+  textType: Extract<TextType, 'note'>;
+  /**
+   * 配下に含むテキストのフォントサイズの抽象値。合わせてtextTypeの指定が必須で、typeに応じた値が指定可能
+   */
+  textSize?: NoteFontSize;
+  /**
+   * 配下に含むテキストの行送りの抽象値（`line-height`）。合わせてtextTypeとtextSizeの指定が必須で、typeに応じた値が指定可能
+   */
+  textLeading?: NoteLeading;
+};
+
+/**
+ * If type is not specified, an empty object is returned because it is unknown how it is to be styled, i.e. it is not styled.
+ * If type is specified but size or leading is not, specify default values (md or default.)
+ */
+export const textStyleVariables = ({
+  type,
+  size: _size,
+  leading: _leading,
+}:
+  | {
+      type: undefined;
+      size: never;
+      leading: never;
+    }
+  | {
+      type: Extract<TextType, 'body'>;
+      size: BodyFontSize | undefined;
+      leading: BodyLeading | undefined;
+    }
+  | {
+      type: Extract<TextType, 'note'>;
+      size: NoteFontSize | undefined;
+      leading: NoteLeading | undefined;
+    }): CSSProperties => {
+  if (type == null) return {};
+
+  const size = type != null && _size == null ? 'md' : _size;
+  const leading = type != null && _leading == null ? 'default' : _leading;
+
+  switch (type) {
+    case 'body':
+      return {
+        '--text-size': size ? cssFontSizeToken({ type, size }) : 'inherit',
```

Comment:
If no text-related prop is specified, the value of `inherit` is specified in the CSS.